### PR TITLE
more pinning

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Change the version numbers to sha values for the python modules

### CODE REVIEW

**Comment by akihikokuroda** [comment]
There are still a few pinning issues remaining.  I'll work on them in the next PR.

File: .github/workflows/client-verify.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
 
       - name: Install tox
         run: |
-          pip install tox==4.16.0
+          pip install git+https://github.com/tox-dev/tox.git@2f7c60ec71d47f4099394af11999a840bc97c9a9 #tox==4.17.0
```
Comment (Tansito) [inline .github/workflows/client-verify.yaml:42]: Why are we installing it from git directly, Aki? I think I would prefer the other approach

Comment (Tansito) [reply .github/workflows/client-verify.yaml:42]: Basically it's my comment in the rest of the places where we are using git

Comment (akihikokuroda) [reply .github/workflows/client-verify.yaml:42]: That's the way to install using the hash value by pip.  The scan tool doesn't like the version nunber.

Comment (psschwei) [reply .github/workflows/client-verify.yaml:42]: I think it may be worth an internal discussion on whether we need to implement all these recommendations

Comment (Tansito) [reply .github/workflows/client-verify.yaml:42]: Agree, I don't know what best practice or security problem are we trying to cover with this to be honest 😅

**Comment by akihikokuroda** [comment]
This seems too much.  I close this.

**Pull request closed without merging**